code/modules/research/xenobiology/crossbreeding/warping

```diff
@@ -0,0 +1,796 @@
+
+
+///it makes people that step on the tile very cold.
+/obj/effect/warped_rune/darkbluespace/proc/cold_tile()
+	for(var/mob/living/carbon/H in T)
+		H.adjust_bodytemperature(-1000) //it sounds like a lot but unless you'r weak to cold it's more like a slight inconvenience
+
+
+
+
+/* makes a rune that absorb food, whenever someone step on the rune the nutrition come back to them, not all of it of course.
+TODO : Improve it so it doesn't steal the spotlight of the chef but instead acts as a direct help to the chef to make his work more convenient
+*/
+
+
+/obj/item/slimecross/warping/silver
+	colour = "silver"
+	effect_desc = "Draw a rune that feeds whoever walks on it, but you need to feed the rune with food too."
+	runepath = /obj/effect/warped_rune/silverspace
+
+
+/obj/effect/warped_rune/silverspace
+	desc = "Feed me and I will feed you back, such is the deal."
+	icon_state = "food_rune"
+	var/nutriment = 0 //Used to remember how much food/nutriment has been absorbed by the rune
+
+/obj/effect/warped_rune/silverspace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/food_time)
+
+
+///eats food, if it ate 5 food it will put anyone who walk on it to a well fed status
+/obj/effect/warped_rune/silverspace/proc/food_time()
+	for(var/obj/item/reagent_containers/food/snacks/S in T) //checks if there's snacks on the rune.and then vores the food
+		playsound(T, 'sound/effects/splat.ogg', 20, TRUE) //TODO : use the munching sound effect or whatever it's called
+		nutriment++
+		qdel(S)
+	for(var/mob/living/carbon/human/H in T)
+		if (nutriment <= 4 || H.nutrition >= (NUTRITION_LEVEL_FULL - 2)) //needs 5 nutriment stack and for the person to not be well fed or fat
+			return
+		H.set_nutrition(NUTRITION_LEVEL_FULL - 1)
+		nutriment -= 5
+
+
+
+/* Bluespace rune, originally called "purespace",reworked so that the last person that walked on the rune will swap place with the next person stepping on it*/
+
+/obj/item/slimecross/warping/bluespace
+	colour = "bluespace"
+	runepath = /obj/effect/warped_rune/bluespace
+	effect_desc = "Draw a rune that will swap the place of the next two person that walk on the rune."
+
+obj/effect/warped_rune/bluespace
+	desc = "Everyone is everywhere at once, yet so far away from each other"
+	icon_state = "bluespace_rune"
+	runepath = /obj/effect/warped_rune/bluespace
+	var/mob/living/carbon/C // first person to run into the rune
+	var/mob/living/carbon/M //second person that run into the rune
+	var/count = 0
+
+obj/effect/warped_rune/bluespace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/swapping)
+
+
+///the first two person that stepped on the rune swap places after the second person stepped on it. TODO: fix the potential lag abyse
+obj/effect/warped_rune/bluespace/proc/swapping()
+	if(count == 0)
+		for(C in T)
+			count++
+			return
+	for(M in T)   //teleports the first person to the rune and the second person to the old location of the first person, prone to lag if abused
```

Comment:
Considering monkeys aren't teleportable by the rune (it only works with humans and its subtypes) I doubt it would be very doable. the last time I did get to make it lag is by basically getting another human against the wall and continuously teleporting each other back and forth by just walking in. Although I never managed to reproduce that properly after a few code tweaks.However if you did manage to get a lot of humanised monkey on the same tile you could probably deal a substantial amount of lag.Which is probably what makes this rune one of the most dangerous to have merged as of now.